app/src/components/datasets/DatasetContentTabs/General/FineTuneButton.tsx

```diff
@@ -327,78 +338,80 @@ const FineTuneModal = ({ disclosure }: { disclosure: UseDisclosureReturn }) => {
                     tab.
                   </Text>
 
-                  <HStack>
-                    <Text fontWeight="bold">Learning rate</Text>{" "}
-                    <InfoCircle
-                      tooltipText="
+                  {advancedConfigEnabled && (
+                    <VStack alignItems="start">
+                      <HStack>
+                        <Text fontWeight="bold">Learning rate</Text>{" "}
+                        <InfoCircle
+                          tooltipText="
 Controls the magnitude of updates to the model's parameters during training."
-                    />
-                  </HStack>
+                        />
+                      </HStack>
 
-                  <VStack w="full" alignItems="flex-start">
-                    <NumberInput
-                      step={0.0001}
-                      min={0.0001}
-                      max={1000}
-                      value={trainingConfigOverrides?.learning_rate}
-                      backgroundColor="white"
-                      onChange={(_, learning_rate) => {
-                        setTrainingConfigOverrides((prevState) => ({
-                          ...prevState,
-                          learning_rate,
-                        }));
-                      }}
-                    >
-                      <NumberInputField
-                        placeholder={trainingConfigOverrides?.learning_rate?.toString() || "Auto"}
-                      />
-                      <NumberInputStepper>
-                        <NumberIncrementStepper />
-                        <NumberDecrementStepper />
-                      </NumberInputStepper>
-                    </NumberInput>
-                  </VStack>
-                  <HStack>
-                    <Text fontWeight="bold">Number of Epochs</Text>{" "}
-                    <InfoCircle tooltipText="The total number of times the entire dataset is passed forward and backward through a neural network during training." />
-                  </HStack>
+                      <NumberInput
+                        step={0.0001}
+                        min={0.0001}
```

Comment:
The minimum here should be 0 actually. It's possible someone will want to train with a learning rate several orders of magnitude smaller than our default.

Reply:
Done!